scripts/build_heat_demand.py

```diff
@@ -1,42 +1,42 @@
+"""Build heat demand time series."""
 
 import geopandas as gpd
 import atlite
 import pandas as pd
 import xarray as xr
-import scipy as sp
-import helper
+import numpy as np
+from helper import clean_invalid_geometries
 
-if 'snakemake' not in globals():
-    from vresutils import Dict
-    import yaml
-    snakemake = Dict()
-    with open('config.yaml') as f:
-        snakemake.config = yaml.safe_load(f)
-    snakemake.input = Dict()
-    snakemake.output = Dict()
+if __name__ == '__main__':
 
-time = pd.date_range(freq='m', **snakemake.config['snapshots'])
-params = dict(years=slice(*time.year[[0, -1]]), months=slice(*time.month[[0, -1]]))
+    if 'snakemake' not in globals():
+        from vresutils import Dict
+        import yaml
+        snakemake = Dict()
+        with open('config.yaml') as f:
+            snakemake.config = yaml.safe_load(f)
+        snakemake.input = Dict()
+        snakemake.output = Dict()
 
-cutout = atlite.Cutout(snakemake.config['atlite']['cutout_name'],
-                       cutout_dir=snakemake.config['atlite']['cutout_dir'],
-                       **params)
+    time = pd.date_range(freq='h', **snakemake.config['snapshots'])
+    cutout_config = snakemake.config['atlite']['cutout']
+    cutout = atlite.Cutout(cutout_config).sel(time=time)
 
-clustered_busregions_as_geopd = gpd.read_file(snakemake.input.regions_onshore).set_index('name', drop=True)
+    clustered_regions = gpd.read_file(
+        snakemake.input.regions_onshore).set_index('name').squeeze()
 
-clustered_busregions = pd.Series(clustered_busregions_as_geopd.geometry, index=clustered_busregions_as_geopd.index)
+    clean_invalid_geometries(clustered_regions)
```

Comment:
Just a comment on this one here: 
The `clean_invalid_geometries` function can be easily shortcutted by set a buffer to the geopandas series. So 
```
    clustered_regions = gpd.read_file(
        snakemake.input.regions_onshore).set_index('name').squeeze().buffer()
```
should work and every thing is done. Also no need to convert it to a pandas series, as the indicatormatrix takes also geopandas series.

Reply:
Great! Is there a case at all where the buffer should not be zero?

Yes, I squeezed the regions into a `gpd.Series` to avoid the `pd.Series` workaround.

Reply:
Don't think so, at least the current helper function would not catch it either.